.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: daily
+    commit-message:
+      # Prefix all commit messages with "[docker] "
+      prefix: "[docker] "
```

Comment:
To follow our conventional commit format, shouldn't it be chore(Dockerfile):  or build(Dockerfile):  ?

Reply:
Good idea, thanks!